tests/expected_report.txt

```diff
@@ -21,7 +21,7 @@ tests/input/LowCaseTypes.php                          2       0
 tests/input/namespaces-spacing.php                    7       0
 tests/input/NamingCamelCase.php                       7       0
 tests/input/new_with_parentheses.php                  18      0
-tests/input/not_spacing.php                           8       0
+tests/input/not_spacing.php                           10      0
```

Comment:
does `$testing =(string) $test;` count as two errors or there's something more that is caught by the sniff in this file?

Reply:
Yes, there's and I couldn't figure out a way to fix it for now. I'll close it and open a new one once I discover what we can do about it :)